### PR TITLE
Update schema.js

### PR DESCRIPTION
**Pull request by ToddWebDev**
# New schema
Update the vets-json-schema for 21P-0969

_Please ensure you have incremented the version in_ `package.json`.

## Pull Requests to update the schema in related repositories
_After you've merged your changes to vets-json-schema you'll need to make PR's to vets-website and vets-api. Please link them here._

- https://github.com/department-of-veterans-affairs/vets-website/pull/32825

### CODE REVIEW

File: src/schemas/21P-0969-kitchen_sink/example.json [hunk 1]
```diff
@@ -23,40 +23,228 @@
...
+        "addedFundsAfterEstablishment": false,
+        "revocable": true,
+        "receivingIncomeFromAnnuity": false,
+        "annualReceivedIncome": 0.0,
+        "canBeLiquidated": true,
+        "surrenderValue": 80000.0
       }
     ]
   }
-}
```
Comment (RachalCassity) [inline src/schemas/21P-0969-kitchen_sink/example.json:62]: Please add newline

File: src/schemas/21P-0969-overflow/example.json [hunk 2]
```diff
@@ -23,47 +23,231 @@
...
+        "addedFundsAfterEstablishment": false,
+        "revocable": true,
+        "receivingIncomeFromAnnuity": false,
+        "annualReceivedIncome": 0.0,
+        "canBeLiquidated": true,
+        "surrenderValue": 80000.0
       }
     ]
   }
-}
```
Comment (RachalCassity) [inline src/schemas/21P-0969-overflow/example.json:69]: Please add new line

File: src/schemas/21P-0969-kitchen_sink/example.json (hunk 1 above)
Comment (ToddWebDev) [reply src/schemas/21P-0969-kitchen_sink/example.json:62]: When I open this file in a code editor, there is a trailing empty line (251) after the new closing curly brace on line 250. Do you want to add another new line(252)?

File: src/schemas/21P-0969-overflow/example.json (hunk 2 above)
Comment (ToddWebDev) [reply src/schemas/21P-0969-overflow/example.json:69]: When I open this file in a code editor, there is a trailing empty line (254) after the new closing curly brace on line 253. Do you want to add another new line(255)?

**Review by mjknight50 — APPROVED** [review]
GTG

**Review by balexandr — APPROVED** [review]

File: src/schemas/21P-0969-kitchen_sink/example.json (hunk 1 above)
Comment (RachalCassity) [reply src/schemas/21P-0969-kitchen_sink/example.json:62]: Sorry, github issue then

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**